.clang-tidy

```diff
@@ -22,6 +22,7 @@ Checks: >-
   -clang-analyzer-nullability.NullablePassedToNonnull,
   -clang-analyzer-nullability.NullReturnedFromNonnull,
   -clang-analyzer-nullability.NullableReturnedFromNonnull,
+  -clang-analyzer-nullability.NullableDereferenced,
```

Comment:
Is this intended for this PR?

Reply:
Yeah. The sysroot for the newer API level is triggering some false-positives in a few places: https://ci.chromium.org/ui/p/flutter/builders/try/Linux%20linux_clang_tidy/6722/overview.